sync.WaitGroup passed by value

<dgryski@kamek[rend] \ʕ◔ϖ◔ʔ/ > staticcheck ./...
/home/dgryski/work/src/cvs/gocode/src/github.com/Netflix/rend/client/sizes.go:43:29: should pass sync.WaitGroup by pointer

--- a/client/sizes.go
+++ b/client/sizes.go
@@ -36,11 +36,11 @@ func main() {
 		prot = t
 	}
 
-	var wg sync.WaitGroup
+	wg := &sync.WaitGroup{}
 	wg.Add(f.NumWorkers)
 
 	for i := 0; i < f.NumWorkers; i++ {
-		go func(prot common.Prot, wg sync.WaitGroup) {
+		go func(prot common.Prot, wg *sync.WaitGroup) {
 			conn, err := common.Connect("localhost", f.Port)
 			if err != nil {
 				panic("Couldn't connect")

